src/features/class-full-page-cache-404.php

```diff
@@ -122,6 +122,10 @@ public function boot(): void {
 	 */
 	public static function action__template_redirect(): void {
 
+		if ( ! apply_filters( 'wp_404_caching_enabled', true ) ) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
So, this won't disable the plugin completely. Cron jobs are still being created. Some hooks are still fired.

Reply:
I thought about that, but this seemed like it was sufficient for what we need in our current use case. What are your thoughts?

Reply:
I'm open to renaming that action in the 404 plugin to something other than "enabled" to make it more accurate to what is happening. I don't really know if a filter or action to disable the plugin makes sense, since the plugin could just be actually disabled in WordPress if someone wants to bypass it entirely. What are you thinking is the best strategy?